stockflux-chart/src/App.js

```diff
@@ -24,12 +25,32 @@ import './styles/BitFlux/notification.css';
 import './styles/app.css';
 import './styles/BitFlux/sprite.css';
 
+let isListening = false;
```

Comment:
Could we use React hooks for `isListening`?

Reply:
Leaving it out as we don't want it to trigger an additional render